apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/file/EnumAsEnumBuilder.kt

```diff
@@ -61,6 +66,27 @@ class EnumAsEnumBuilder(
     return TypeSpec.companionObjectBuilder()
         .addProperty(typePropertySpec())
         .addFunction(safeValueOfFunSpec())
+        .addFunction(knownValuesFunSpec())
```

Comment:
This is so that we can do `SomeEnum.Companion.knownValues()` for both enums as enums and enums as sealed classes.